recipes/lytekit/meta.yaml

```
@@ -0,0 +1,89 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lytekit-{{ version }}.tar.gz
+  sha256: bf3d9e127ad02b94e88c9b7f4f8df2e81946104c470c935d00ad99b312f46915
+
+build:
+  skip: true  # [py<37 or win or py>310]
+  entry_points:
+    - pyflyte-execute=flytekit.bin.entrypoint:execute_task_cmd
+    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
+    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
+    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
+    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - lyteidl ==0.2.1 =pyhd8ed1ab_1
```

I was testing this right after release, and the CI wasn't picking up the lates build, so this worked for me.  I'll change it back an see what happens

```suggestion
    - lyteidl ==0.2.1
```